open-ce/build_image.py

```diff
@@ -67,15 +67,19 @@ def _validate_input_paths(local_conda_channel, conda_env_file):
     if not os.path.exists(local_conda_channel) or not os.path.exists(conda_env_file):
         raise OpenCEError(Error.INCORRECT_INPUT_PATHS)
 
-    # Check if local conda channel path is subdir of the docker build context
+    # Check if local conda channel path is subdir of the container build context
     if not utils.is_subdir(local_conda_channel, os.path.abspath(BUILD_CONTEXT)):
         raise OpenCEError(Error.LOCAL_CHANNEL_NOT_IN_CONTEXT)
 
-def build_runtime_docker_image(args):
+def build_runtime_container_image(args):
     """
     Create a runtime image which will have a conda environment created
     using locally built conda packages and environment file.
     """
+    if args.container_tool:
+        global CONTAINER_TOOL  # pylint: disable=W0603
```

Comment:
Probably shouldn't be using a global variable anymore. The container_tool argument should be passed around to whichever functions need it. The global variable was only for constants.